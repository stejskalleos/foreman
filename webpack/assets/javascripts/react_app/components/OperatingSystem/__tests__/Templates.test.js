import React from 'react';
import { Provider } from 'react-redux';

import store from '../../../redux';
import { testComponentSnapshotsWithFixtures } from '../../../common/testHelpers';

import OperatingSystemTemplates from '../Templates';

const fixtures = {
  'should render': {}
}

describe('OperatingSystemTemplates', () => {
  testComponentSnapshotsWithFixtures(() => (
    <Provider store={store}>
      <OperatingSystemTemplates />
    </Provider>
  ),
  fixtures
  );
});
