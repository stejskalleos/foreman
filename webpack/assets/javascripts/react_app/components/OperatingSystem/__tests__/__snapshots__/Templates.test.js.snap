// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OperatingSystemTemplates should render 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(Symbol.observable): [Function],
    }
  }
>
  <OperatingSystemTemplates />
</Provider>
`;
